discovery-provider/src/app.py

```diff
@@ -83,55 +76,6 @@ def get_eth_abi_values():
 
 
 def init_contracts():
-    registry_address = web3.toChecksumAddress(shared_config["contracts"]["registry"])
```

Comment:
@isaacsolo confirming we want to remove this

Reply:
yup looks good